textBasedAdventure.js

```diff
@@ -1,5 +1,335 @@
-const readline1 = require('readline-sync')
+const readLineSync = require('readline-sync')
 
-let nameInput = readline.question("Enter your name: ")
+let nameInput = readLineSync.question("Enter your name: ")
+
+const ageVerification = () =>{
+    console.log('This game is for players that are 18 and over because this story gets pretty graphic')
+    let ageQuest = Number (readLineSync.question("How old are you?"))
+    if(ageQuest >= 18){
+        console.clear()
+        startGame()
+    }else if(ageQuest < 18){
+        console.log("Sorry you're too young! So Scram!")
+        quitGame()
+    }else if(!ageQuest){
+        console.clear()
+        console.log("Not a valid number my friend")
+        ageVerification()
+    }
+}
+const play = () => {
+    console.log(`Hello ${nameInput}!  Welcome to Survival Island.`)
+    console.log('The rules of the game are simple. You will be presented with options choose the correct one and you will advance, otherwise you will not make it to the end.' )
+    if(readLineSync.keyInYNStrict("Sound good? ")) {
+        console.clear()
+        ageVerification()
+        startGame();
+    } else {
+        quitGame();
+    }
+  }
+  const startGame = () => {
+      console.log("Welcome to what may be your last day standing")
+      console.log("You awake on a beach. It's cold and dark. You reach into your pocket and find a lighter as well as a match.")
+      let decision1 = readLineSync.question("Which one would you like to use? Match or Lighter?")
```

Comment:
You should update your users input for better user experience. 
I want to be able to type in MATCH, match, or any form of MaTch